mods/ctf/ctf_map/ctf_map_core/schem_map.lua

```diff
@@ -335,7 +335,7 @@ local function place_map(map)
 		local res = minetest.place_schematic(map.pos1, schempath,
 				map.rotation == "z" and "0" or "90")
 
-		assert(res)
+		assert(res, "Unable to place schematic, does the MTS file exist? path=" .. schempath)
```

Comment:
IMHO, this will make it look more user-friendly. 🙂
```suggestion
		assert(res, "Unable to place schematic, does the MTS file exist? Path: " .. schempath)
```